src/routes/v2/authenticatedSites/media.ts

```diff
@@ -108,22 +108,20 @@ export class MediaRouter {
     { directoryName: string },
     { directories: MediaDirOutput[] },
     never,
-    { page: number },
+    never,
```

Comment:
hey was this intentional, why we making this never ah?

Reply:
we do not do any pagination/slicing here